interfaces/PA-App/src/app/helpers/openInfoPopup.ts

```diff
@@ -0,0 +1,13 @@
+import { InfoPopupComponent } from '../info-popup/info-popup.component';
+import { PopoverController } from '@ionic/angular';
+
+export async function openInfoPopup(popoverController: PopoverController, message: string) {
```

Comment:
This should move into the MoreInfoButton-component as a 'local' method.

The `message` would be a `@Input()`-property that you can pass into the `popoverController` then